R/txtq.R

```diff
@@ -113,6 +147,17 @@ R6_txtq <- R6::R6Class(
         quote = FALSE
       )
     },
+    txtq_reset = function(){
+      unlink(private$db_file, force = TRUE)
```

Comment:
what happens if something gets pushed between unlink and file creation?

Reply:
`$txtq_reset()` is a private method, so I am not worried. Users are forced to call the API-level `$reset()` method, which uses `$txtq_exclusive()` to prevent this sort of race condition.